### ISSUE TITLE
gitlab-ci sonar error

### ISSUE THREAD

**Issue by loverto** [gabrie-allaigre/sonar-gitlab-plugin#41]
56444 [ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.3.0.603:sonar (default-cli) on project bksx-springmvc: Failed to execute project builder: com.talanlabs.sonar.plugins.gitlab.CommitProjectBuilder: Unable to perform GitLab WS operation: http://130.10.8.210/api/v3/projects/40/repository/commits/CI_COMMIT_SHA/comments -> [Help 1]

**Comment by gabrie-allaigre** [comment]
Hi....
`-Dsonar.gitlab.commit_sha=$CI_COMMIT_SHA`

**Comment by loverto** [comment]
Thanks ，But I found that the problem has been changed, and now the problem is?

75271 [INFO] Report status=failed, desc=SonarQube reported 205 issues, with 30 critical (fail) and 88 major and 82 minor and 5 info

**Comment by loverto** [comment]
gitlab version 9.2.1  sonarqube version 6.3.1

**Comment by gabrie-allaigre** [comment]
By default, plugin fail if critical issue > 0 (here 30 critical (fail) )

**Comment by loverto** [comment]
So how do i set it up?

**Comment by gabrie-allaigre** [comment]
By properties sonar.gitlab.max_blocker_issues_gate, sonar.gitlab.max_critical_issues_gate (show README)
or by SonarQube -> Project -> Administration -> General Settings -> GitLab

**Comment by loverto** [comment]
SonarQube -> Project -> Administration -> General Settings -> GitLab    no way ?

**Comment by gabrie-allaigre** [comment]
![image](https://cloud.githubusercontent.com/assets/14994297/26670200/10dbcd10-46b1-11e7-85e5-bba2a812b2be.png)

**Comment by loverto** [comment]
version 2.0.1

**Comment by gabrie-allaigre** [comment]
?

**Comment by loverto** [comment]
images not upload 。。。。。  
My interface is not the same for you？
Google Chrome problem ？

**Comment by gabrie-allaigre** [comment]
In SonarQube, project settings Administration -> General Settings -> GitLab
Image is ok (refresh ?)

**Comment by loverto** [comment]
![sonar](https://cloud.githubusercontent.com/assets/3992018/26670921/32b3d5a6-46e6-11e7-9fcb-3ccc0bb5943a.png)

**Comment by gabrie-allaigre** [comment]
No, use general settings **for Project**, no global general settings

**Comment by loverto** [comment]
Where is the report?

**Comment by gabrie-allaigre** [comment]
Mail or in commit in gitlab

**Comment by gabrie-allaigre** [comment]
ok ?

**Comment by loverto** [comment]
did not find

**Comment by gabrie-allaigre** [comment]
In GitLab Repository -> Commits -> change branch and check comment on commit
